### ISSUE TITLE
reid accuracy

### ISSUE THREAD

**Issue by CarlHuangNuc** [open-mmlab/mmtracking#531]
Why retrain reid config alway get very low accuracy(mAP:50),

From your log & pretrain model, it can achieve about 96% mAP.

**Comment by dyhBUPT** [comment]
Hi, we can't understand your question accurately.
We recommend using the [issue template](https://github.com/open-mmlab/mmtracking/tree/master/.github/ISSUE_TEMPLATE).

Best wishes.

**Comment by biyoml** [comment]
I had the same issue. Did you solve it?

**Comment by dyhBUPT** [comment]
Can your retrained ReID model achieve the same MOT metrics for Tracktor?
I mean, you just replace the reid ckpt with your retrained results and leave detector and ohter hyper-parameters unchanged.

**Comment by biyoml** [comment]
Although my retrained ReID checkpoint only had ~60% mAP during training, the Tracktor (tracktor_faster-rcnn_r50_fpn_4e_mot17-private-half) using my checkpoint can achieve 63.7% MOTA on MOT17.

**Comment by JinZhangYu** [comment]
same problem here, any fix now?

**Comment by JinZhangYu** [comment]
https://download.openmmlab.com/mmtracking/fp16/reid_r50_fp16_8x32_6e_mot17_20210731_033055.log.json
This log shows each epoch has 8000 iterations under GPU8, but master tools/dataset_converters/mot/mot2reid.py cannot generate so many crop images under MOT17.

I use 4 GPUs, and each epoch only has 873 iterations.
